src/Connections/GithubRepo.ts

```diff
@@ -649,8 +649,8 @@ export class GitHubRepoConnection extends CommandConnection<GitHubRepoConnection
         }
     }
 
-    @botCommand("assign", "Assign an issue to a user", ["number", "...users"], [], true)
-    public async onAssign(userId: string, number: string, ...users: string[]) {
+    @botCommand("assign", "Assign an issue to a user. If number is ommitted, the latest issue is used. If users is omitted, you are assigned.", [], ["number", "...users"], true)
```

Comment:
How about backticking the parameter names to make them jump out better?
```suggestion
    @botCommand("assign", "Assign an issue to a user. If `number` is ommitted, the latest issue is used. If `users` is omitted, you are assigned.", [], ["number", "...users"], true)
```
I've confirmed that it works:
![image](https://user-images.githubusercontent.com/3271094/199288999-3d3cc53a-9462-4af0-a3bc-b524449f7e81.png)